bedrock/settings/static_media.py

```diff
@@ -526,6 +526,32 @@
         ),
         'output_filename': 'css/firefox_new_way_of_the_fox-bundle.css',
     },
+    'firefox_new_lifestyle_base': {
+        'source_filenames': (
+            'css/newsletter/fxnewsletter-subscribe.less',
+            'css/firefox/family-nav.less',
```

Comment:
Don't think we need these two files, as the family nav and newsletter signup aren't displayed on these pages.